src/core.js

```diff
@@ -150,6 +150,19 @@ extend( QUnit, {
 
 registerLoggingCallbacks( QUnit );
 
+function storeFixture() {
+
+	// Avoid overwriting user-defined values
+	if ( hasOwn.call( config, "fixture" ) ) {
+		return;
+	}
+
+	var fixture = id( "qunit-fixture" );
```

Comment:
Seeing this here, I'm pretty sure I was wrong to suggest moving it. `storeFixture` really does belong in reporter/html.js, but still benefits from the new `hasOwn` guard.